### PR TITLE
[2.8][Translator] Added missing changelog notes.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | ~ |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Component/Translation/CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,10 @@ CHANGELOG
 2.8.0
 -----
 
- * deprecated Translator::getMessages(), use TranslatorBagInterface::getCatalogue() instead.
- * added options 'as_tree', 'inline' to YamlFileDumper
+ * deprecated Translator::getMessages(), rely into TranslatorBagInterface::getCatalogue() instead.
```
Comment (Tobion) [inline src/Symfony/Component/Translation/CHANGELOG.md]: typo: rely on

Comment (aitboudad) [inline src/Symfony/Component/Translation/CHANGELOG.md]: fixed, thanks!

File: src/Symfony/Component/Translation/Translator.php [hunk 2]
```diff
@@ -290,11 +290,11 @@ protected function getLoaders()
      *
      * @return array[array] indexed by catalog
      *
-     * @deprecated since version 2.8, to be removed in 3.0. Use TranslatorBagInterface::getCatalogue() method instead.
+     * @deprecated since version 2.8, to be removed in 3.0. Rely on TranslatorBagInterface::getCatalogue() method instead.
```
Comment (Tobion) [inline src/Symfony/Component/Translation/Translator.php]: We always use `Use ... instead` in deprecation messages. I don't think this change is worth.

Comment (aitboudad) [inline src/Symfony/Component/Translation/Translator.php]: it was suggested by stof see https://github.com/symfony/symfony/pull/14546#discussion_r29592981

Comment (Tobion) [inline src/Symfony/Component/Translation/Translator.php]: Then we would need to change it in many places. Many things are not drop-in replacements. Will be pretty hard to change it everywhere. Thus not worth.

Comment (aitboudad) [inline src/Symfony/Component/Translation/Translator.php]: it's ok for me, reverted!

**Comment by Tobion** [comment]
Thank you @aitboudad.

**Pull request merged**